src/flamegraph.js

```diff
@@ -27,6 +22,44 @@ export default function () {
   var searchSum = 0
   var totalValue = 0
   var maxDelta = 0
+  var p = partition()
+
+  const containerElement = document.createElement('div')
+  const titleElement = document.createElement('div')
+  const nodesSpaceElement = document.createElement('div')
+  const nodesElement = document.createElement('div')
+  const tipElement = document.createElement('div')
+  // Safari is very annoying with its default tooltips for text with ellipsis.
+  // The only way to disable it is to add dummy block element inside.
+  const tipDeterringElement = document.createElement('div')
+  containerElement.className = 'd3-flame-graph'
+  titleElement.className = 'title'
+  nodesSpaceElement.className = 'nodes-space'
+  nodesElement.className = 'nodes'
+  tipElement.className = 'tip'
+  nodesSpaceElement.appendChild(nodesElement)
+  nodesSpaceElement.appendChild(tipElement)
+  containerElement.appendChild(titleElement)
+  containerElement.appendChild(nodesSpaceElement)
+
+  const externalState = {
```

Comment:
This is being used keep the tooltip in the same place and selecting text only, correct? If that's the case, I would try to move that to a separate feature pull request. Not a big fan of adding document event listeners and handlers in the plugin. There might be a way of keeping the key state in an external object.

Reply:
I can't think of a solution right now, that's why I suggested taking a separate PR to deal with it.